packages/twenty-server/src/engine/metadata-modules/flat-field-metadata/utils/compare-two-flat-field-metadata.util.ts

```diff
@@ -32,6 +32,15 @@ export const flatFieldMetadataEntityJsonbProperties = [
 export type FlatFieldMetadataEntityJsonbProperties =
   (typeof flatFieldMetadataEntityJsonbProperties)[number];
 
+export const relationFlatFieldMetadataPropertiesToCompare = [
+  'label',
+  'description',
+  'isActive',
+] as const satisfies FlatFieldMetadataPropertiesToCompare[];
+
+export type RelationFlatFieldMetadataPropertiesToCompare =
```

Comment:
or in a type if the plan is to only export the type

Reply:
Same as above